front/app/containers/Admin/sideBar/index.tsx

```diff
@@ -277,8 +285,19 @@ class Sidebar extends PureComponent<
             <MenuItem navItem={navItem} key={navItem.name} />
           ))}
           <Spacer />
+
+          <MenuLink
+            href={formatMessage(messages.linkToAcademy)}
+            target="_blank"
+          >
+            <IconWrapper>
+              <Icon name="academy" />
+            </IconWrapper>
+            <Text>{formatMessage({ ...messages.academy })}</Text>
```

Comment:
I think functionally it's the same but you don't have to spread the message with `formatMessage` (you do with the React component, like `<FormattedMessage {...messages.academy} />`)